service/matching/matchingEngine.go

```diff
@@ -327,14 +327,19 @@ pollLoop:
 			})
 			if err != nil {
 				// will notify query client that the query task failed
-				e.deliverQueryResult(task.query.taskID, &queryResult{err: err})
+				if err := e.deliverQueryResult(task.query.taskID, &queryResult{err: err}); err != nil {
+					e.logger.Error("failed to delivery query result", tag.Error(err))
+				}
```

Comment:
@andrewjdawson2016 can you take a look?

Reply:
please make this `_ := e.deliverQueryResult(...)` there is no need to log in this case and the method already emits a metric.

Reply:
The error from `deliverQueryResult` never actually needs to be used. Let's remove the return error from this method instead of logging each time we get an error. I expect this case to happen fairly often and the log will not be helpful.

Reply:
Good call. Updated